core-services/prow/02_config/openshift/assisted-image-service/_prowconfig.yaml

```diff
@@ -31,30 +31,7 @@ tide:
   merge_method:
     openshift/assisted-image-service: squash
   queries:
-  - includedBranches:
-    - release-ocm-2.4
-    - release-ocm-2.5
-    - release-ocm-2.6
-    labels:
-    - approved
-    - bugzilla/valid-bug
-    - lgtm
-    missingLabels:
-    - backports/unvalidated-commits
-    - bugzilla/invalid-bug
-    - do-not-merge/hold
```

Comment:
don't we need those? hold and wip

Reply:
a duplicated entry basically, so now configurations for master and ``release-ocm-*`` are unified